### DISCUSSION TITLE
Bambulab Region Lock

### DISCUSSION THREAD

**Discussion by lunDreame** [greghesp/ha-bambulab#356] [Q&A]
I bought a Chinese domestic product and am using it in Korea. Currently, I installed and linked the Chinese Bambu Handy through a Chinese number. However, I want to replace the Chinese version with a global ROM. Does Bambu divide China/Global into serials? As far as I know, in the case of the early P1S model, it was released only as a global version. Then, I think it is divided into software. So I think I can download global software and work on the OTA firmware for my Chinese domestic use. Can you help me?

**Comment by AdrianGarside** [comment]
I haven't heard of any way to drive firmware updates locally. As far as I know a China locked device is permanently China locked.

**Comment by AdrianGarside** [comment]
Maybe the newly announce X1+ firmware replacement will work.
